salt/metalk8s/kubeadm/init/init.sls

```diff
@@ -1,3 +1,4 @@
 include:
   - .preflight
   - .kubelet-start
+  - .certs
```

Comment:
So we want to call the `ca` state? Can we call it directly instead?

Reply:
Depends of the comment thread just above